setup.py

```diff
@@ -268,6 +268,7 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
     'click==6.7',
     'flake8>=3.6.0',
     'flake8-colors',
+    'flake8-isort',
```

Comment:
> License: GNU General Public License v2 (GPLv2) (GPL version 2)

I can't remember what the outcome was on this for other devel only changes. @mik-laj Do you remember?

(Somewhat annoyingly isort itself is MIT)

Reply:
Discussion about flake-isort in Airflow is available here:
https://github.com/apache/airflow/pull/4892#issuecomment-472369397